src/modules/workspaces/workspaces.controller.ts

```diff
@@ -152,11 +155,136 @@ export class WorkspacesController {
     return this.workspaceUseCases.getWorkspaceTeams(user, workspaceId);
   }
 
-  @Get('/:workspaceId/teams/:teamId/members')
+  @Get('/teams/:teamId/members')
+  @ApiOperation({
+    summary: 'Gets team members',
+  })
+  @ApiOkResponse({
+    description: 'Members of the team',
+  })
+  @UseGuards(WorkspaceGuard)
+  @WorkspaceRequiredAccess(AccessContext.WORKSPACE, WorkspaceRole.MEMBER)
+  async getTeamMembers(@Param('teamId') teamId: WorkspaceTeamAttributes['id']) {
+    if (!teamId) {
+      throw new BadRequestException('Invalid team ID');
+    }
+    return this.workspaceUseCases.getTeamMembers(teamId);
+  }
+
+  @Patch('/teams/:teamId')
+  @ApiOperation({
+    summary: 'Edits team data',
+  })
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    description: 'Team has been edited',
+  })
+  @UseGuards(WorkspaceGuard)
+  @WorkspaceRequiredAccess(AccessContext.TEAM, WorkspaceRole.MANAGER)
+  async editTeam(
+    @Param('teamId') teamId: WorkspaceTeamAttributes['id'],
+    @Body() editTeamBody: EditTeamDto,
+  ) {
+    if (!teamId || !isUUID(teamId)) {
```

Comment:
Do we have some validator for this? I remember that we had something to validate UUIDs and is used on the DTOs so we may be able to use it here.